frontend/tests/fixtures/standard.sql

```diff
@@ -285,12 +289,19 @@ values ((select id from realms where full_path = '/love'), 'kiwi', 'Kiwis', 'alp
 insert into realms (parent, path_segment, name, child_order)
 values ((select id from realms where full_path = '/love'), 'turtles', 'Turtles', 'alphabetic:desc');
 
+insert into realms (parent, path_segment, name)
+values ((select id from realms where full_path = '/moon'), 'finanzamt', 'Finanzamt'); 
```

Comment:
The test data cracks me up :D

The commit message still says that a subrealm is added to "support" page, but the changes say otherwise. Commit message title also has a typo. Maybe you want to fix that, but not a blocker.